images/libvirt-kubevirt/qemu-kube

```diff
@@ -93,24 +93,8 @@ set -e
 # to adhere to the resource limitations of the container.
 exec sudo -C 10000 unshare --mount bash -s << END
 
-  function _term() {
-    rm -rf /etc/resolv.conf.\$\$
-    pkill -P \$! --signal SIG\$1
-  }
-
-  trap "_term TERM" TERM
```

Comment:
The traps should be redundant - k8s/docker will kill every process in the namespace which includes both the qemu-kube wrapper & real QEMU. Now we no longer need to cleanup the resolv.conf file, we don't need to catch the signals